specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -36,13 +36,63 @@ model AcsUserDisconnectedEventData {
   userCommunicationIdentifier: CommunicationIdentifierModel;
 }
 
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.CallStarted event. */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallStartedEventData
+  extends AcsCallingEventProperties {
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.CallEnded event. */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallEndedEventData
+  extends AcsCallingEventProperties {
+  /** The communication identifier of the user who was disconnected */
+  endedBy?: AcsCallingEventEndedByProperties;
+
+  /** The reason for the call ended event. */
+  reason?: AcsCallingEventReasonProperties;
+
+  /** Duration of the call in seconds. */
+  callDurationInSeconds?: float64;
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.CallParticipantAdded event. */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallParticipantAddedEventData
+  extends AcsCallParticipantEventProperties {
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.CallParticipantRemoved event. */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallParticipantRemovedEventData
+  extends AcsCallParticipantEventProperties {
+}
+
 /** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.ChatMessageReceived event. */
 #suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
 model AcsChatMessageReceivedEventData
   extends AcsChatMessageEventBaseProperties {
   ...AcsChatMessageBaseEventData;
 }
 
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.ChatAzureBotCommandReceivedInThread event. */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsChatAzureBotCommandReceivedInThreadEventData
+  extends AcsChatMessageEventInThreadBaseProperties {
```

Comment:
Why is azureBotCommand extending from different model vs typing indicator event extending a different one? Shouldn't the schema be same for both?

Reply:
what is the resolution here?